pom.xml

```diff
@@ -301,11 +317,6 @@
             <artifactId>opentelemetry-exporter-otlp</artifactId>
             <version>${opentelemetry.version}</version>
         </dependency>
-        <dependency>
-            <groupId>io.opentelemetry</groupId>
-            <artifactId>opentelemetry-exporter-otlp-http-trace</artifactId>
-            <version>${opentelemetry.version}</version>
-        </dependency>
```

Comment:
OTEL 1.19.0 version does consolidate OTLP exporters under single artifact - `opentelemetry-exporter-otlp`. So removing this non-existent dependency for 1.19.0.